Projects/App/Sources/UI/Organization/VerifyCode/View/VerifingCodeViewController.swift

```diff
@@ -0,0 +1,218 @@
+//
+//  VerifingCodeViewController.swift
+//  App
+//
+//  Created by 김태호 on 2022/11/02.
+//  Copyright (c) 2022 com.zesty. All rights reserved.
+//
+
+import Combine
+import UIKit
+import DesignSystem
+import SnapKit
+
+final class VerifingCodeViewController: UIViewController {
+    
+    // MARK: - Properties
+    private var cancelBag = Set<AnyCancellable>()
+    
+    private let isSE: Bool = UIScreen.main.isHeightLessThan670pt
+    
+    private var keyboardEndFrameHeight: CGFloat?
+    
+    private lazy var titleView = MainTitleView(title: "이메일로 받은 코드를\n알려주세요", subtitle: "\(userEmail)", hasSymbol: true)
+    
+    private let warningMessage = UILabel()
+    private let otpStackView = OTPStackView()
+    private let timerLabel = UILabel()
+    
+    private let resendStackView = UIStackView()
+    private let resendLabel = UILabel()
+    private let resendEamilButton = UIButton(type: .custom)
+    
+    private let arrowButton = ArrowButton(initialDisable: false)
+    
+    // TODO: ViewModel로 옮길 것들입니다
+    let userEmail = "mingming@pos.idserve.net"
+    var isArrowButtonHidden: Bool = true
+    var isCodeValid: Bool = true
+    var userInputCode: String = ""
+    var timer = "03:00"
+    
+    // MARK: - LifeCycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureUI()
+        createLayout()
+        bindUI()
+    }
+    
+    // MARK: - Function
+    
+    @objc func resendButtonTapped() {
+        print("버튼 눌림")
+        showToastMessage()
+    }
+    
+    private func showToastMessage() {
+        let toastLabel = UILabel()
+        toastLabel.backgroundColor = .black
+        toastLabel.textColor = .white
+        toastLabel.font = .systemFont(ofSize: 16.0, weight: .regular)
+        toastLabel.textAlignment = .center
+        toastLabel.text = "메일을  다시 보냈어요"
+        toastLabel.layer.cornerRadius = 25
+        toastLabel.clipsToBounds  =  true
+        toastLabel.numberOfLines = 0
+        
+        self.view.addSubview(toastLabel)
+        
+        toastLabel.snp.makeConstraints { make in
+            make.centerX.equalToSuperview()
+            make.top.equalTo(view.snp.top).offset(-50)
```

Comment:
p4;

이거 그냥 toastLabel의 bottom을 view.snp.top에 맞추면 되지 않나요?
이렇게 하는게 toastLabel의 높이가 변경됐을 때 수정할 곳이 하나가 줄어서 더 낫다고 생각합니다!

```suggestion
            make.bottom.equalTo(view.snp.top)
```

Reply:
오 그 생각은 못했네요! 
수정했습니다 ! 68e59759c17a26db1eaaecf055e79a3fc7b178fa

Reply:
코드를 지울 때 이상한 부분을 해결하였습니다! 
텍스트필드가 비어있는 경우에 대한 예외처리가 안되어있어서 추가하였습니다!
68e59759c17a26db1eaaecf055e79a3fc7b178fa